app/routes/app/brainstorm/$marketplace.c.new.tsx

```diff
@@ -1,18 +1,18 @@
 import { Select, TextInput, Textarea, NativeSelect, NumberInput, Text, Title, Button } from "@mantine/core";
 import { DatePicker, TimeInput } from "@mantine/dates";
 
-export default function CreateTopic() {
+export default function CreateChallenge() {
   const marketplace = true;
 
   return (
     <div className="mx-auto max-w-3xl space-y-10">
       <div className="grid grid-cols-1 md:grid-cols-3 gap-5 items-center">
         <Title order={2} weight={600} className="md:col-span-3">
-          {"Create Topic"}
+          {"Create Challenge"}

```

Comment:
Static text doesn't need to be in `{""}`s. There's a few instances in this file that can be cleaned up.

`{}` is only necessary when you're injecting a dynamic value. e.g: `<Text>{challenge.title}</Text>`

Reply:
oops lol didn't even notice this when I edited it. Did a find and replace I think. Will fix